docs/content/stable/secure/tls-encryption/tls-authentication.md

```diff
@@ -178,9 +178,9 @@ SSL connection (protocol: TLSv1.2, cipher: ECDHE-RSA-AES256-GCM-SHA384, bits: 25
 Type "help" for help.
 ```
 
-### TLS with password and certificate authentication
+### TLS with password authentication and certificate verification
 
-This configuration requires the client to use client-to-server encryption and authenticate with both the appropriate certificate and the password to connect.
+This configuration requires the client to use client-to-server encryption, supplying a signed client certificate, while using a password to authenticate. Note that the server verifies that the client certificate is signed by the configured CA but it does not use the CN (Common Name) of the client certificate to authenticate the user. It is not possible to configure the server to require authentication via both password and client certificate.
```

Comment:
should these changes also go in preview

Reply:
Good point. In preview, I will add a note talking about verify-ca, just in case someone is interested. In general, I suspect people use TLS + password + no client certs or  TLS + client certs + no password so we don't need a separate section for clientcert=verify-ca